packages/typo3-guides-extension/src/Command/RunDecorator.php

```diff
@@ -0,0 +1,150 @@
+            if ($value === null) {
+                continue;
+            }
+
+            $options['--' . $option] = $value;
+        }
+
+        $arguments = $input->getArguments();
+        if ($arguments['input'] === null) {
+            $guessedInput = $this->guessInput($output);
+        }
+
+        if ($arguments['output'] === null) {
+            $arguments['output'] = getcwd() . '/' . self::DEFAULT_OUTPUT_DIRECTORY;
+        }
+
+        $input = new ArrayInput(
+            [
+                ... $arguments,
+                ... $options,
+                ... $guessedInput ?? [],
+            ],
+            $this->getDefinition()
+        );
+
+        return $this->innerCommand->execute($input, $output);
+    }
+
+    public function getDescription(): string
+    {
+        return $this->innerCommand->getDescription();
+    }
+
+    public function getHelp(): string
+    {
+        return $this->innerCommand->getHelp();
+    }
+
+    public function setApplication(Application $application = null): void
+    {
+        parent::setApplication($application);
+        $this->innerCommand->setApplication($application);
+    }
+
+    /** @return mixed[] */
+    public function getUsages(): array
+    {
+        return $this->innerCommand->getUsages();
+    }
+
+    public function getNativeDefinition(): InputDefinition
+    {
+        return $this->innerCommand->getNativeDefinition();
+    }
+
+    public function getSynopsis(bool $short = false): string
+    {
+        return $this->innerCommand->getSynopsis($short);
+    }
+
+    public function getDefinition(): InputDefinition
+    {
+        return $this->innerCommand->getDefinition();
+    }
+
+    public function mergeApplicationDefinition(bool $mergeArgs = true): void
+    {
+        $this->innerCommand->mergeApplicationDefinition($mergeArgs);
+    }
+
+    /** @return array<string, string> */
+    private function guessInput(OutputInterface $output): array
+    {
+        $currentDirectory = getcwd();
+        if ($currentDirectory === false) {
+            return [];
+        }
+
+        $inputDirectory = $currentDirectory . '/Documentation';
+
+        if (file_exists($inputDirectory)) {
+            if ($output->isVerbose()) {
+                $output->writeln(sprintf('Input directory not specified, using %s', $inputDirectory));
+            }
+
+            foreach (self::EXTENSIONS as $extension) {
+                foreach (self::INDEX_FILE_NAMES as $index) {
+                    if (file_exists($inputDirectory . DIRECTORY_SEPARATOR . $index . '.' . $extension)) {
+                        return [
+                            'input' => $inputDirectory,
+                            '--input-format' => $extension,
+                        ];
+                    }
+                }
+            }
+        }
+
+        if ($output->isVerbose()) {
+            $output->writeln('Index documetation file not found, trying README.rst or README.md');
```

Comment:
```suggestion
            $output->writeln('Index documentation file not found, trying README.rst or README.md');
```

Maybe add a new class constant FALLBACK_FILE_NAMES with README.rst and README.md as content. This way we have one place and can use implode on that array for the message. Imagine we allow later also Readme.md and Readme.rst. Just adjust the array and everything is fine.